.github/workflows/tests.yml

```diff
@@ -23,6 +23,11 @@ jobs:
           lfs: true
           fetch-depth: 0
 
+      - name: Use Node.js Own version
```

Comment:
It was picking up version 18 and failing to build

Reply:
can you adjust the same for the other 2 actions?

Reply:
Updated @sbe-arg